build/repo.props

```diff
@@ -75,8 +75,7 @@
                     Exclude="
                       @(ProjectToExclude);
                       $(RepositoryRoot)**\bin\**\*;
-                      $(RepositoryRoot)**\obj\**\*;
-                      $(RepositoryRoot)**\AutobahnTestApp\**\*;" />
```

Comment:
@mikaelm12 I think this may incidentally fix Autobahn test issues (at least until we move to Helix...but then everything is going to break...)